{
  "uid" : "2e50eafc05423496",
  "name" : "Home page - smoke test",
  "fullName" : "tests.ui.test_search.TestWeb#test_homepage_title13",
  "historyId" : "4c6a26788916d97b0c591def5d62e104",
  "time" : {
    "start" : 1649270051243,
    "stop" : 1649270052148,
    "duration" : 905
  },
  "status" : "passed",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "_session_faker",
    "time" : {
      "start" : 1649270043746,
      "stop" : 1649270043831,
      "duration" : 85
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "browser",
    "time" : {
      "start" : 1649270050337,
      "stop" : 1649270050337,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "web_setup",
    "time" : {
      "start" : 1649270050337,
      "stop" : 1649270051242,
      "duration" : 905
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "web_setup::0",
    "time" : {
      "start" : 1649270052149,
      "stop" : 1649270052274,
      "duration" : 125
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.usefixtures('web_setup')"
  }, {
    "name" : "parentSuite",
    "value" : "tests.ui"
  }, {
    "name" : "suite",
    "value" : "test_search"
  }, {
    "name" : "subSuite",
    "value" : "TestWeb"
  }, {
    "name" : "host",
    "value" : "fv-az201-861"
  }, {
    "name" : "thread",
    "value" : "3495-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.ui.test_search"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 3,
        "skipped" : 0,
        "passed" : 1,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "671ec5eee2bb2384",
        "reportUrl" : "https://rupu1906.github.io/docker-selenium/41//#testresult/671ec5eee2bb2384",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f43ada42170>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1649193829742,
          "stop" : 1649193829742,
          "duration" : 0
        }
      }, {
        "uid" : "de171ede74f74294",
        "reportUrl" : "https://rupu1906.github.io/docker-selenium/39//#testresult/de171ede74f74294",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f2a6a766200>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1649193204538,
          "stop" : 1649193204538,
          "duration" : 0
        }
      }, {
        "uid" : "17cf4ecc9a11bd15",
        "reportUrl" : "https://rupu1906.github.io/docker-selenium/38//#testresult/17cf4ecc9a11bd15",
        "status" : "broken",
        "statusDetails" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='chrome', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f81bd586170>: Failed to establish a new connection: [Errno 111] Connection refused'))",
        "time" : {
          "start" : 1649183949127,
          "stop" : 1649183949127,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ "@pytest.mark.usefixtures('web_setup')" ]
  },
  "source" : "2e50eafc05423496.json",
  "parameterValues" : [ ]
}